app/src/js/components/Table/Table.js

```diff
@@ -48,10 +50,19 @@ const List = ({
   const [selected, setSelected] = useState([]);
   const [clearSelected, setClearSelected] = useState(false);
   const [page, setPage] = useState(1);
-
+  let sortBy;
+  if (tableID) {
+    sortBy = sorts[tableID];
+  }
+  let idMatch;
+  if (!sortBy || sortBy.length === 0) {
+    idMatch = initialSortId;
+  } else {
+    idMatch = sortBy[0].id;
```

Comment:
We don't just want the first sortBy entry, there could be multiple. We should be passing the full sortBy object to `buildSortKey`. Also worth noting that we can't be sure that desc is always true for `sortBy[0].id`, so this wouldn't work in that case

Reply:
I don't understand. How can there be multiple values that you can sortBy, when you change the value that you are sorting by, doesn't it just get replaced? Also why does it being descending matter in this case because we aren't checking for that? I am just a little confused

Reply:
`sortBy` is an array generated by `react-table` and stored in the state (and which we're retrieving here). It looks like this `Array<Object<id: columnId, desc: Bool>>`. There can be multiple objects in this array because `react-table` supported multi column sort.

`initialSortId` is different because it is just defining the default column we want to sort by. This is why we pass that id to `[{ id: initialSortId, desc: true }]` so that we're constructing an array in the same format that the table component is expecting. `sortBy` is already in the correct format, so we don't need to do anything to it.

Reply:
Got it, pushing changes again